backend/src/controllers/banner.controller.js

```
@@ -1,33 +1,75 @@
+      try {
+        validateRelativeUrl(url, "url");
+      } catch (err) {
+        return res.status(400).json({ errors: [{ msg: err.message }] });
+      }
     }
 
     const colorFields = { fontColor, backgroundColor };
-    const colorCheck = checkColorFieldsFail(colorFields, res)
-    if (colorCheck) { return colorCheck };
+    const colorCheck = checkColorFieldsFail(colorFields, res);
+    if (colorCheck) {
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)